support-frontend/assets/components/adFreeSection/adFreeSection.scss

```diff
@@ -1,61 +1,114 @@
-.component-ad-free-section {
-  font-family: $gu-text-egyptian-web;
+@import '~stylesheets/gu-sass/gu-sass';
 
-  border-top: 1px solid gu-colour(neutral-86);
-  border-bottom: 1px solid gu-colour(neutral-86);
+[class^="ad-free-section"] {
+  box-sizing: border-box;
 }
 
-.component-ad-free-section__content {
-  box-sizing: border-box;
-  padding-left: 10px;
-  padding-right: 10px;
-  padding-top: 2px;
+.ad-free-section, .ad-free-section-c {
   width: 100%;
-  position: relative;
-  border-left: 1px solid gu-colour(neutral-86);
+  
+  @include mq($from: phablet) {
+    display: flex;
+    max-width: gu-span(12);
+    border-left: 1px solid gu-colour(neutral-86);
+  }
 }
 
-.component-ad-free-section__header {
-  width: 100%;
-  @include gu-fontset-heading;
-  margin-bottom: 8px;
+.ad-free-section__content {
+  padding-left: 10px;
 
-  @include mq($from: leftCol) {
-    width: 620px;
-    line-height: 1.17;
+  @include mq($from: phablet) {
+    width: 190%;
+    padding-bottom: 40px;
+  }
+
+  @include mq($from: desktop) {
+    width: 100%;
+    padding-bottom: 80px;
   }
 }
 
-.component-ad-free-section__col {
-  width: 100%;
-  padding-right: 40px;
-  @include mq($from: leftCol) {
-    width: 440px;
+.ad-free-section-c__content {
```

Comment:
Shall we rename any reference to `.ad-free-section-c__*` to `.ad-free-section__*` in here and the component?

There's a few references to `.ad-free-section-b__*` in the `.scss` that can be removed now too.

Reply:
Yes good spot